### ISSUE TITLE
Add source CQL files to repo? (ELM JSON files are already here)

### ISSUE THREAD

**Issue by mcjustin** [uwcirg/asbi-screening-app#1]
This repo (and its parent) includes the ELM JSON files (https://github.com/uwcirg/asbi-screening-app/tree/master/src/cql), but not the source CQL files. 

https://cds.ahrq.gov/cdsconnect/artifact/alcohol-screening-using-usaudit-alcohol-use-disorders-identification-test
"Logic Files" section at bottom has a "UsAuditLogicFiles.zip" link pointing at https://cds.ahrq.gov/sites/default/files/cds/artifact/logic/2021-05/UsAuditLogicFiles.zip . This zip includes UsAuditLogicLibrary.cql, which at first glance (cross-matching the string `HasRecentAlcoholScreeningObservations`\*, seems likely to be the source for the elm/json).

\* Note: this string is also found in ELM files in the intervention app: https://github.com/asbi-cds-tools/asbi-intervention-app/tree/master/src/cql

**Comment by achen2401** [comment]
For **WhoAuditLogicLibrary.json**, the cql source appears to be at https://cds.ahrq.gov/cdsconnect/artifact/alcohol-screening-using-world-health-organization-who-alcohol-use-disorders (see zip file at the end)
For **NidaQsToUsAuditLogicLibrary.json** the cql source appears to be at https://cds.ahrq.gov/cdsconnect/artifact/alcohol-and-other-substance-use-screening-using-national-institute-drug-abuse (see zip file at the end)
For **FHIRHelpers-4.0.1.json** the cql source appears to be at https://build.fhir.org/ig/WorldHealthOrganization/smart-anc/Library-FHIRHelpers.html

**Comment by mcjustin** [comment]
Nice @achen2401 , I was just wondering why the first cql file I referenced seemed specific to just one of the questionnaires.

**Comment by mcjustin** [comment]
Just as a check of tools and artifacts, I used https://github.com/cqframework/clinical_quality_language/blob/master/Src/java/cql-to-elm/OVERVIEW.md to translate these 3 CQL files that @achen2401  found to ELM/json (per Maggie's how-to synopsis [here](https://docs.google.com/document/d/1EOVhwEzEwXlMWY9SDQjNxTLR6OOrKQUdo41TE9KCyS4/edit)), in order to compare w/ what is in this repo (`*AuditLogicLibrary.json`).
Two of them matched perfectly, except for whitespace in 2 lines.  However, `WhoAuditLogicLibrary.json` had many more lines in the repo version - I expect that the zip file we found for that was slightly out of date. No worries, the WHO variant is not of much interest at this point, if ever. FYI @ivan-c @achen2401
